### PR TITLE
fix(jsonschema): fix job validation

### PR DESCRIPTION
**Pull request by rclsilver**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-cds#20216.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20216.1 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20216.1 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Review by yesnault — APPROVED** [review]

**Review by fsamin — APPROVED** [review]

File: engine/api/v2_jsonschema.go [hunk 1]
```diff
@@ -72,8 +72,8 @@ func getWorkerModelNames(ctx context.Context, db gorp.SqlExecutor, u *sdk.AuthUs
 	}
 	wmNames := make([]string, 0, len(entities))
 	for _, wm := range entities {
-		shortRef := strings.TrimPrefix(strings.TrimPrefix(wm.Ref, sdk.GitRefBranchPrefix), sdk.GitRefTagPrefix)
-		wmNames = append(wmNames, fmt.Sprintf("%s/%s/%s/%s@%s", wm.ProjectKey, wm.VCSName, wm.RepoName, wm.Name, shortRef))
+		wmNames = append(wmNames, fmt.Sprintf("%s/%s/%s/%s@%s", wm.ProjectKey, wm.VCSName, wm.RepoName, wm.Name, wm.Ref))
```
Comment (richardlt) [inline engine/api/v2_jsonschema.go:75]:
```suggestion
		wmNames = append(wmNames, fmt.Sprintf("%s/%s/%s/%s@%s", wm.ProjectKey, wm.VCSName, wm.RepoName, wm.Name, wm.Ref))
		shortRef := strings.TrimPrefix(strings.TrimPrefix(wm.Ref, sdk.GitRefBranchPrefix), sdk.GitRefTagPrefix)
		wmNames = append(wmNames, fmt.Sprintf("%s/%s/%s/%s@%s", wm.ProjectKey, wm.VCSName, wm.RepoName, wm.Name, shortRef))
```

**Review by richardlt — CHANGES_REQUESTED** [review]
Allow short ref usage

**Comment by ovh-cds** [comment]
CDS Report build-cds#20221.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20221.0 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✘  
  * Standalone Tests v2 ✔

**Review by richardlt — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20221.1 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**